skiko/src/commonMain/kotlin/org/jetbrains/skia/Image.kt

```diff
@@ -388,6 +425,10 @@ private external fun Image_nGetImageInfo(ptr: NativePointer, imageInfo: InteropP
 @ModuleImport("./skiko.mjs", "org_jetbrains_skia_Image__1nMakeShader")
 private external fun Image_nMakeShader(ptr: NativePointer, tmx: Int, tmy: Int, samplingModeVal1: Int, samplingModeVal2: Int, localMatrix: InteropPointer): NativePointer
 
+@ExternalSymbolName("org_jetbrains_skia_Image__1nMakeRawShader")
+@ModuleImport("./skiko.mjs", "org_jetbrains_skia_Image__1nMakeRawShader")
+private external fun Image_nMakeRawShader(ptr: NativePointer, tmx: Int, tmy: Int, samplingModeVal1: Int, samplingModeVal2: Int, localMatrix: InteropPointer): NativePointer
```

Comment:
It's common code, so JNI/C++ code in `skiko/skiko/src/jvmMain/cpp/common
/Image.cc` is missing but required

Reply:
added to JVM